src/sql/ImportExportOptions.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SQLPARSER_IMPORT_EXPORT_OPTIONS_H
+#define SQLPARSER_IMPORT_EXPORT_OPTIONS_H
+
+namespace hsql {
+
+// Name unchanged for compatibility.
```

Comment:
I don't get this comment.

Reply:
Dates back to when we added the `COPY` statement and, with it, the ability to export tables. Even though we use these file types for import and export, we kept the name so people with forks don't have to rename stuff in their code.

Reply:
This thinking is also the reson why I added the encoding as another member of the statement rather than just passing the options object with both file type and encoding.

Reply:
What name would you have chosen?

Reply:
For the enum? In the grammar, we simply call it file type (so `FileType` as enum name):
https://github.com/hyrise/sql-parser/blob/9a67d2470ba053c9f9c63ebca7c100e6f73f0d2c/src/parser/bison_parser.y#L457
I remember that it was a big discussion back then and some insisted on compatibility. Anyway, I think it's kinda late for that now.

Reply:
Can you briefly mention that we call it FileyType in the grammar, but won't adapt it here?
Not stating what the alternative is (nobody is aware of the probably old discussion) does not give away why the current name might not be perfect. :)

Reply:
Feel free to zünd afterwards.